CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Subpackages for the available recommender types
```

Comment:
I'll leave that decision to you and Martin. You can then either change it in this PR or Martin can take care of it during his upcoming refactoring

Reply:
one issue is that `hybrid`, `composite` and `meta` mostly describe the same things, ie an object that is patched out of several similar objects, they do have slightly different falvors though

For the renaming of strategies I prefer `MetaStrategy` because theres an analogy to a meta study, a study that considers several other study objects, selecting between them. Theres no flavor of combining different recommenders for the different subspaces here, in both `composite` and `hybrid` I do see that flavor, hence my fav here is `meta`

How about we then rename the `NaiveHyrbidRecommender` into `NaiveHybridSpaceRecommender` so that it makes clear hybrid here goes over the two subspaces and is not about selecting recommenders

Reply:
+1 for renaming the `NaiveHybridRecommender` as this just makes it clearer what is meant here. Does this change where we want to put it?

Reply:
so then the name for the previously-strategies is the only thing left to decide, these renamings dont have to be in this PR, so lets rather keep this PR as is (also because theres already another based on this), decide in the baybathon and Ill change it in the upcomings refactorings

plz upvote or resolve